### ISSUE TITLE
Icons location

### ISSUE THREAD

**Issue by guccisekspir** [boeledi/flutter_reactive_button#3]
when longtap button ,icons appear on the right of the screen . What should I do

**Comment by shrijanRegmi** [comment]
yeah it happens with me too. Basically when I put it inside a Row widget, it happens. I don't know what to do. Please help
